test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -3501,6 +3501,52 @@ public void Mesh2dGetNodeEdgeDataThroughApi()
                 }
             }
         }
+
+
+        [Test]
+        public void Mesh2dMakeTwoRectangularGridThroughApi()
```

Comment:
It is not clear to me what exactly this test tests. Add a function header comment, or change the test to illustrate it better. For example, the first and second mesh have exactly the same parameters. When the second has a different set of parameters you can check that they are indeed different.